tests/lib/desktop-portal.sh

```diff
@@ -25,6 +25,13 @@ BusName=org.freedesktop.impl.portal.spread
 ExecStart=/usr/bin/python3 $TESTSLIB/fakeportalui/portalui.py
 EOF
     mkdir -p /usr/share/xdg-desktop-portal/portals
+    # Disable any existing portal implementations
+    for p in /usr/share/xdg-desktop-portal/portals/*.portal; do
+        if [ ! -f "$p" ]; then
```

Comment:
Hmm I thought that `nullglob` is set, but maybe I'm misremembering.

Reply:
To be honest, I added this out of old muscle memory writing shell scripts.  It doesn't look like we have `nullglob` set though, yeah.